tasks/prime.py

```diff
@@ -228,6 +228,52 @@ def update_mto_shipment(self):
         if success:
             self.replace_prime_data(PrimeObjects.MTO_SHIPMENT, mto_shipment, resp)
 
+    @tag(PrimeObjects.MTO_SHIPMENT.value, "updateMTOShipmentAddress")
+    @task
+    def update_mto_shipment_address(self):
+        move_task_order = self.get_random_data(PrimeObjects.MOVE_TASK_ORDER)
```

Comment:
You should be able to grab an MTO Shipment directly instead of needing an MTO and then creating a new shipment, if you wanted! Is there a particular reason you did it this way instead?

Reply:
No - this was an oversight on my part! Thanks so much! I've updated it :)